### ISSUE TITLE
Name of custom filter don't save after adding the same one with new name 

### ISSUE THREAD

**Issue by SoLenny** [AdguardTeam/AdGuardForSafari#239]
We need to decide what should do AdGuard when user adding custom filter which is already installed.
Error or maybe something else?

**Comment by Mizzick** [comment]
I agree, it would be better to update existing instance with new properties such as name, trusted flag etc.

**Comment by Mizzick** [comment]
Works as expected.
Adding custom filter with the same url (url is an identifier for these filters) results in updating filter properties: name and trusted
